Test/ContentPipeline/IntermediateDeserializerTest.cs

```diff
@@ -537,5 +537,16 @@ public void GenericTypes()
                 Assert.AreEqual(4.2f, genericTypes.B.Value);
             });
         }
+
+        [Test]
+        public void DeserialiseDefaults()
+        {
+            DeserializeCompileAndLoad<DeserialiseDefaults>("25_DeserialiseDefaults.xml", deserialiseDefaults =>
+            {
+                Assert.AreEqual(Vector2.Zero, deserialiseDefaults.EmptyVector2);
+                Assert.AreEqual(Vector2.Zero, deserialiseDefaults.Empty2Vector2);
+
```

Comment:
Yeah... we do asserts not only to find current bugs, but to catch future ones.

Like if someone decides to return the wrong default for `Matrix`.